docs/developer-guide/running-locally.md

```diff
@@ -31,7 +31,14 @@ kubectl -n argocd scale deployment/argocd-redis --replicas 0
 
 ### Start local services
 
-Before starting local services, make sure you are present in `argocd` namespace. When you use the virtualized toolchain, starting local services is as simple as running
+Before starting local services, make sure you are present in `argocd` namespace. 
```

Comment:
> Before starting local services, make sure you are present in `argocd` namespace. 

In case people are running it in different namespace, suggest rephrase as:

The started services assume you are running in the namespace where Argo CD is installed. You can set the current context default namespace as follows: